WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -91,6 +98,54 @@ private void ValidateCoordinatorUri(IValidationErrors errors)
 		Settings.CoordinatorUri = coordinatorUri;
 	}
 
+	private void ValidateMaxAllowedCoordinationFeeRate(IValidationErrors errors)
+	{
+		var maxAllowedCoordinationFeeRate = MaxAllowedCoordinationFeeRate;
+
+		if (string.IsNullOrEmpty(maxAllowedCoordinationFeeRate))
+		{
+			return;
+		}
+
+		if (!decimal.TryParse(maxAllowedCoordinationFeeRate, out var maxAllowedCoordinationFeeRateDecimal))
+		{
+			errors.Add(ErrorSeverity.Error, "Invalid number.");
+			return;
+		}
+
+		if (maxAllowedCoordinationFeeRateDecimal > 1)
+		{
+			errors.Add(ErrorSeverity.Error, "Absolute maximum allowed coordination fee rate is 1%");
```

Comment:
And minimum should be 1s/vb as you verify in the config file, otherwise users can enter here 0 and wasabi will fail to load next time.